lib/Remote/HttpCommandExecutor.php

```diff
@@ -210,6 +210,14 @@ public function __construct($url, $http_proxy = null, $http_proxy_port = null)
         $this->setRequestTimeout(30000);
         $this->setConnectionTimeout(30000);
     }
+    
+    /**
+     * Allows to specify the curl options
+     * @param $options
```

Comment:
```suggestion
     * @param array $options
```
https://www.php.net/manual/en/function.curl-setopt-array.php